### ISSUE TITLE
Suggestion for the fuzzy search: moves files already opened on top

### ISSUE THREAD

**Issue by cecton** [jmacdonald/amp#133]
I've started discovering and using amp for 2 days now and I have a few suggestions I would like to share. Here is the most important one:

## TL;DR

In the fuzzy search result: first display the files that are already opened in a buffer and match the result, then the rest.

When the user is annoyed to see a file they are not working on anymore in the search result, they could simply close the buffer.

Alternative: a fuzzy search dedicated to the opened buffers.

## Long Story

Coming from ViM I was used to use the tabs to change of file/buffer. Unfortunately it's not very handy when you have a lot of them and I discovered with Amp that this mechanism can be totally replaced by the fuzzy search of Amp.

BUT there is one problem: on a very large project there are more redundancy in the name of the files. Because of that it's hard to switch of buffer easily in Amp. It would be very nice if we could have a different way to order the items in the fuzzy search: first put the files that are already opened in a buffer, then the rest.

I think it would be the same than we have on browser: you type something in the URL bar, you get result from the search engine but also always from your bookmarks. I use that feature a lot.

**Comment by ls42** [comment]
[fzf.vim](https://github.com/junegunn/fzf.vim/commit/1ef72b14ccd05fdbdb01d253b91a74c4760ae655#commands) has a `:Buffers` command that does something like you mentioned @jmacdonald . It opens a modal with a list of all open buffers and then lets you fuzzy find all open buffers by name.
